bellbot_gui/package.xml

```diff
@@ -1,43 +1,19 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_gui</name>
-  <version>0.0.0</version>
+  <version>0.0.7</version>
   <description>Bellbot GUI</description>
 
   <maintainer email="y.gatsoulis@leeds.ac.uk">Yiannis Gatsoulis</maintainer>
 
   <license>MIT</license>
 
-  <author email="n.a.hawes@cs.bham.ac.uk">Nick Hawes</author>
-  <author email="c.j.c.burbridge@cs.bham.ac.uk">Chris Burbridge</author> 
+  <author email="y.gatsoulis@leeds.ac.uk">Yiannis Gatsoulis</author>
+  <author email="sc12jbmt@leeds.ac.uk">Jawad Tayyub</author> 
+  <author email="j.young@cs.bham.ac.uk">Jay Young</author> 
 
   <buildtool_depend>catkin</buildtool_depend>
-  
-<!--
-  <build_depend>rospy</build_depend>
-  <build_depend>strands_webserver</build_depend>
-  <build_depend>std_srvs</build_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>actionlib</build_depend>
-  <build_depend>actionlib_msgs</build_depend>
-  <build_depend>rosbridge_server</build_depend>
-  <build_depend>strands_webtools</build_depend>
-  <build_depend>bellbot_action_server</build_depend>
-  -->
-
- <!--
-  <run_depend>rospy</run_depend>
-  <run_depend>strands_webserver</run_depend>
-  <run_depend>std_srvs</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>actionlib</run_depend>
-  <run_depend>actionlib_msgs</run_depend>
-  <run_depend>rosbridge_server</run_depend>
-  <run_depend>strands_webtools</run_depend>
-  <run_depend>twitter_bootstrap</run_depend>
-  <run_depend>python-webpy</run_depend>
-  -->
-  
+    
   <build_depend>twitter_bootstrap</build_depend>
   <build_depend>strands_webtools</build_depend>
   <build_depend>strands_webserver</build_depend>
```

Comment:
I think you are missing a few dependencies like `rospy` and `roslib` for example. Please make sure that everything you import is defined as a build and run dependency.